domains/ai.json

```diff
@@ -0,0 +1,14 @@
+{
+    "description": "for ai helpful website and project . ",
+    "owner": {
+        "repo": "https://github.com/ramanandkrgupta/",
+        "email": "ramanand@myyahoo.com",
+
+    },
+
+    "record": {
+        "NS": ["aron.ns.cloudflare.com", "decker.ns.cloudflare.com"],
```

Comment:
You can't use Cloudflare DNS with our domain, since we aren't in the Public Suffix List (PSL) yet.